test-e2e/smoketest.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+
+const WAIT = 1000;
+const NODE_PORT = process.env.NODE_PORT || 3000;
+
+
+module.exports = {
+  'Smoketest' (browser) {
+    browser
+      .url(`http://localhost:${NODE_PORT}/test-data`)
+      .waitForElementVisible('body', WAIT)
+      .assert.containsText('body', 'Books')
```

Comment:
It is a simplest smoketest - just making sure at least something is rendered on a page.

If we want some better testing, then of course we can add more things here, like actual clicking on button and checking rendered list.